app/Controllers/Http/ProfileController.ts

```diff
@@ -0,0 +1,72 @@
+// import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import Profile from "App/Models/Profile"
+import UserProfileValidator from "App/Validators/UserProfileValidator"
+
+export default class ProfileController {
+
+    public async view(){
+        const profiles=await Profile.all()
+        return profiles
+    }
+
+    public async showProfile({params,auth}){
+
+        try{
+            const profile=await Profile.findBy("mobile",params.mobile)
+            const details={}
+
+            const dob=profile.$attributes.dob.c.year+"-"+profile.$attributes.dob.c.month+"-"+profile.$attributes.dob.c.day
+            const date = new Date(dob);
+            const year = date.getFullYear();
+            const month = date.toLocaleString('default', { month: 'short' });
+            const day = date.getDate();
+            const formattedDate = `${year} ${month} ${day}`;
+
+            details.name=profile.$attributes.name
```

Comment:
why can't we use `details.name=profile.name`

Reply:
profile.name will give undefined....
profile object has been defined like this only. Inside,,profile $attributes key is there, where all the details are defined